reflect_test.go

```diff
@@ -742,3 +744,40 @@ func TestPassthrough(t *testing.T) {
 		}
 	})
 }
+
+func Test_reflectNil(t *testing.T) {
+	tt(t, func() {
+		test, vm := test()
+		{
+			vm.Set("func", func(s *_mnoStruct) bool {
+				return s == nil
+			})
+
+			test(`
```

Comment:
I see. The starting ` line was space indented instead of tab. I wonder why go fmt didn't fix it... Anyway, going to fix both issues now.